lib/api/v1/controllers/keywords_test.go

```diff
@@ -66,4 +67,70 @@ var _ = Describe("KeywordsController", func() {
 			})
 		})
 	})
+
+	Describe("GET /keywords", func() {
+		AfterEach(func() {
+			test.CleanUpDatabase()
+		})
+		Context("Given VALID payload", func() {
+			It("returns the status OK", func() {
+				user := fabricators.FabricateTester()
+				payload := map[string]interface{}{
+					"limit": "1",
+				}
+				ctx, resp := test.MakeAuthenticatedRequest(http.MethodGet, "/keywords", nil, payload, user)
+
+				controller := controllers.KeywordsController{}
+				controller.List(ctx)
+
+				Expect(resp.Code).To(Equal(http.StatusOK))
+			})
+
+			It("returns correct response body", func() {
```

Comment:
```suggestion
			It("returns the keywords that belong to the current user, serialized in JSON", func() {
```

Any action returns the correct response body, therefore this description does not add value (and if so, we could just remove it... or improve it so value is provided for the readers).